marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobDataManager.java

```diff
@@ -0,0 +1,66 @@
+package com.marklogic.hub.job;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.dataservices.ExecCaller;
+import com.marklogic.client.dataservices.IOEndpoint;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.hub.HubConfig;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class JobDataManager {
+    ObjectNode apiObj;
+    String apiPath = "/ml-modules/root/data-hub/5/data-services/bulk/deleteJobs.api";
+    ObjectMapper mapper = new ObjectMapper();
+    HubConfig hubConfig;
+
+    JobDataManager(HubConfig hubConfig) {
```

Comment:
Use HubClient instead of HubConfig here; HubConfig is appropriate when we need access to a HubProject. HubClient is much lighter-weight and is good for when you only need to connect to ML.